ci/patch_bazel_windows/compile.yml

```
@@ -17,8 +17,9 @@ jobs:
       CACHE_KEY="$(find ci/patch_bazel_windows -type f -print0 | sort -z | xargs -r0 md5sum | md5sum | awk '{print $1}')"
       TARGET="patch_bazel_windows/bazel-$CACHE_KEY.zip"
       TARGET_URL="https://daml-binaries.da-ext.net/$TARGET"
+      STATUS="$(curl -Is "$TARGET_URL" | head -1 | awk '{print $2}')"
```

You can also make `curl` just print the status code:
```suggestion
      STATUS="$(curl -Is -w '%{http_code}' -o /dev/null "$TARGET_URL")"
```

---

Thanks for pointing that out, I was not aware of that option. I'm not going to include that in this PR, though, as I want this to be a minimal change to fix the issue from yesterday.